dace/transformation/dataflow/wcr_conversion.py

```diff
@@ -69,15 +73,14 @@ def can_be_applied(self, graph, expr_index, sdfg, permissive=False):
             if any(e.src is not me and not isinstance(e.src, nodes.AccessNode)
                    for e in graph.in_edges(me) + graph.in_edges(tasklet)):
                 return False
-            if graph.in_degree(inarr) > 0:
```

Comment:
This will be problematic and may create race conditions if state fission is not performed